openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py

```diff
@@ -0,0 +1,69 @@
+""" Management command to update is_prerequisite_courses_passed flag in ApplicationHub """
+import logging
+import sys
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from django.db.models import Count, Q
+
+from lms.djangoapps.grades.api import CourseGradeFactory
+from openedx.adg.lms.applications.models import ApplicationHub
+from openedx.adg.lms.course_meta.models import CourseMeta
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Updates is_prerequisite_courses_passed flag in ApplicationHub.
+    """
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        pre_req_courses = CourseMeta.open_pre_req_courses.all()
+
+        if not pre_req_courses:
+            sys.exit('Exiting!!! No open pre-req courses found. But there must be some pre-reqs. Please add in admin.')
+
+        users_to_be_checked_for_update = self.get_minimal_users_to_be_checked_for_update(pre_req_courses)
+        for user in users_to_be_checked_for_update:
+            for pre_req_course in pre_req_courses:
+                if self.is_user_failed_in_course(user, pre_req_course):
+                    logger.info('{username} has not yet passed all the pre-reqs'.format(username=user.username))
+                    break
+            else:
+                user_application_hub, _ = ApplicationHub.objects.get_or_create(user=user)
+                user_application_hub.set_is_prerequisite_courses_passed()
+                logger.info(
+                    '{username} has successfully completed the pre-reqs, flag is updated in application hub'.format(
+                        username=user.username
+                    ))
+
+    def get_minimal_users_to_be_checked_for_update(self, pre_req_course):
+        """
+        Filter minimal set of users who should be checked for the update.
+        Params:
+            pre_req_course: List
+        Return:
+            QuerySet
+        """
```

Comment:
Docstring format is different. Please update other occurrences as well.
```
"""
Filter minimal set of users who should be checked for the update.

Args:
    pre_req_course (CourseMeta): List of pre req courses

Returns:
    A querySet for users who should be checked for the update
"""
```

By the way name `pre_req_course` suggests it is a single course, not a list.


**Note:** To auto-generate docstring template, you can got to pycharm settings->tools->Python integrated tools -> Docstring format -> Goolge